tor/strings/en_US.yml

```diff
@@ -81,6 +81,10 @@ responses:
       Sorry; I can't find your transcript post on the link. Please message the mods to have them look at this.
 
       Alternatively, did you remember to put the footer in your comment? _Make sure_ that you're copying and pasting the footer as it's shown, complete with all the special characters!
+      
+      If the footer is correct, have you tried replying "Done" again after one-two minutes? Sometimes, freshly posted transcriptions are not seen by the bot, so waiting some minutes should fix the issue.
+      
+      If it still doesn't work, it could be the sign of another bug, and a Modmail will warn the mods pretty quickly!
```

Comment:
"another bug, and a Modmail will warn the mods pretty quickly!" -> "another bug; let us know by [sending a Modmail](https://www.reddit.com/message/compose?to=%2Fr%2FTranscribersOfReddit&subject=Transcription%20Issue&message=) so we can take a look!"